src/components/OnboardingLoader.js

```diff
@@ -3,11 +3,11 @@ import OnboardingModal from './Onboarding/OnboardingModal'
 
 function OnboardingLoader({ children }) {
   const [onboardingCompleted, setOnboardingCompleted] = useState(
-    localStorage.getItem('onboardingCompleted')
+    localStorage.getItem('onboardingCompleted') === 'true'
```

Comment:
just curious why is that?

Reply:
Just to be explicit (also goes with [this change](https://github.com/aragon/court-dashboard/pull/76/files/5d31943cda7dff6a90b2687ca9229ce21e7ba720..229953d4cab4e62d92d5f2eff0ad3f5860513602#diff-1e528074d9943161292be4b97cad109aR10)). This is because `localStorage` stores strings only, so `true` was converted into `"true"` implicitly.